README.md

```diff
@@ -49,6 +49,48 @@ If you'd like to protect your Sentry install with SSL/TLS, there are
 fantastic SSL/TLS proxies like [HAProxy](http://www.haproxy.org/)
 and [Nginx](http://nginx.org/). You'll likely want to add this service to your `docker-compose.yml` file.
 
+OR do the following things to enable auto redirect to https behind a reverse proxy
+
+Uncomment the following in `docker-compose.yml` nginx directive
+```
+ #expose:
+ #  - '443'
+ ports:
+ #  - '443:443'
+   - '$SENTRY_ONPREMISE_PORT:80/tcp'
+```
+
+And uncomment the following in your sentry.config.py
+```
+SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+SESSION_COOKIE_SECURE = True
+CSRF_COOKIE_SECURE = True
+SOCIAL_AUTH_REDIRECT_IS_HTTPS = True
+SENTRY_USE_SSL = True
+```
+
+And uncomment the following `#`s in your nginx config, and simply replace the contents of the `cert.crt` and `cert.key` in the ssl folder found inside the nginx folder
```

Comment:
Same concern again, let's not repeat config in the readme, just use markers instead.

Reply:
resolved